docs/CarbonData-on-docker.md

```diff
@@ -0,0 +1,72 @@
+#  CarbonData on docker
+This tutorial provides an easy way to try CarbonData. To follow along with this guide, please install [docker](https://www.docker.com/products/docker) at first.
+
+## 1 Building carbondata
+To get the jar files, we need build carbondata project. Here Spark version is 1.6.1 and Hadoop version is 2.7.2.
+```shell
+git clone https://github.com/apache/incubator-carbondata.git carbondata
```

Comment:
Now we don't release a packaged carbon version.  Because we can reuse exists maven repository in local machine, so I use maven to build carbon in local machine. In docker image, we need to download many dependence jars from Apache maven repository again. 

Better to download a packaged carbon version directly.   After release version, we can skip maven building, and just need to use wget to download the packaged carbon version in docker image.